app/frontend/src/internationalization/trans/chefs/ar/ar.json

```diff
@@ -762,10 +762,15 @@
     "createOnlineTitle": "قم بإنشاء نماذج عبر الإنترنت لجمع المعلومات من عملائك وتحسين سير العمل الخاص بك.",
     "getStarted": "البدء"
   },
+  "baseStepper": {
+    "designForm": "نموذج التصميم",
+    "setUpForm": "قم بإعداد النموذج",
+    "publishForm": "نشر النموذج",
```

Comment:
Looks like `publishForm` got left in here - good to remove it if we don't need it.